app/src/test/kotlin/com/wire/android/ui/WireActivityViewModelTest.kt

```diff
@@ -569,6 +570,7 @@ class WireActivityViewModelTest {
             every { observeSyncStateUseCaseProviderFactory.create(any()).observeSyncState } returns observeSyncStateUseCase
             every { observeSyncStateUseCase() } returns emptyFlow()
             coEvery { observeIfAppUpdateRequired(any()) } returns flowOf(false)
+            coEvery { observeNewClients() } returns flowOf()
```

Comment:
Can we add a test verifying that the newly added client dialog will be shown if a new device is observed from the current session and other account?